docs/output.md

```diff
@@ -120,16 +145,23 @@ given barcodes/antibodies.
 
 </details>
 
-This step uses the `pixelator single-cell collapse` command.
+### Compute connected components
 
-The `collapse` command removes duplicate reads and performs error correction.
-This is achieved using the unique pixel identifier and unique molecular identifier sequences to check for
-uniqueness, collapse and compute a read count. The command generates a QC report in JSON format.
-Errors are allowed when collapsing reads if `--algorithm` is set to `adjacency` (this is the default option).
+This step uses the `pixelator single-cell graph` command.
+The input is the edge list parquet file generated in the collapse step and after filtering it
+by count (`--graph_min_count`), the connected components of the graph (graphs) are computed and
```

Comment:
```suggestion
by count (`--graph_min_count`). These molecules will form the edges of the connected components of the graph. When graphs are computed and identified,
```

Reply:
The `and after filtering it by count` section does not make sense anymore with the edits.
Could you add the complete paragraph in one suggestion?

Reply:
```
The input is the edge list parquet file generated in the collapse step. These molecules are filtered by count (`--graph_min_count`) to form the edges of the connected components of the graph. When graphs are computed and identified, their ID names are added back to the edge list in a column called "component".
```